### ISSUE TITLE
net_https sample vpk opens then closes. net_http vpk sample crashes.

### ISSUE THREAD

**Issue by jimbob4000** [vitasdk/samples#90]
Hi, I'm having issues with the Http and Https samples on a clean vitasdk build (28-12-2024).

[net_https](https://github.com/vitasdk/samples/tree/master/net_https)
The installed VPK opens, and then closes without doing anything.

[net_http](https://github.com/vitasdk/samples/tree/master/net_http)
The installed VPK doesn't open, it creates a core dump file.

Would you mind testing to see if they work for you please?

Background info:
I've come here to check the samples as there seems to be issues with Lua Player Plus Vita's network file download functions and I think this issue may be related to vitasdk.

Many thanks

**Comment by isage** [comment]
net_http uses url that no longer supports http-only. replace `http://barooney.com/` with `http://neverssl.com/online/` (ant it's the one that crashes)

net_https works as intended. it doesn't provide any output on screen.

Both those samples use sce libs and api

**Comment by jimbob4000** [comment]
Thanks for the quick reply.

You're correct, I got the descriptions the wrong way around, I've updated the issue title and my first comment.

net_http

- The app no longer crashes with your new URL 'http://neverssl.com/online/ '.
- An index.html file is generated (empty file)

net_https

- I though a 'net_https_git_pr.json' file would be created either on ux0 or uma0?

I'll close the issue, thanks for your help
